stdlib/ballerina-database/src/main/ballerina/sql/actions.bal

```diff
@@ -54,8 +54,14 @@ public type CallerActions object {
         The batchUpdate operation implementation for SQL connector to batch data insert.
 
         P{{sqlQuery}} SQL statement to execute
-        R{{}} An `int[]` array of updated row count by each of statements in batch and
-                else `error` will be returned if there is any error
+        R{{}} An `int[]` - The elements in the array returned by the operation may be one of the following  or else
+         an`error` will be returned if there is any error.
```

Comment:
I think it would be much better to align this line with the **An** of the previous line to improve clarity.